### PR TITLE
feat: hint part

### PR DESCRIPTION
**Pull request by Annie04082020**
Redo the hint part due to the socket method update

### CODE REVIEW

File: app/src/utils/session.js
Comment (Annie04082020) [inline app/src/utils/session.js:1]: ref([ ])

Comment (Annie04082020) [inline app/src/utils/session.js:1]: Delete if

File: app/src/components/HintPart.vue [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
   <font-awesome-icon :icon="['far', 'lightbulb']" class="text-yellow-200 p-1" />
   <h1 class="inline text-text font-xl">Hint</h1>
   <div id="hint">
-    <p class="text-text">--Hint--</p>
+    <li v-for="hint in sessionManager.hints.value" :key="hint.id">
+      <p class="text-text">
+        {{ hint }}
+        <br />
+      </p>
+    </li>
```
Comment (lancatlin) [inline app/src/components/HintPart.vue:14]: Implement using pager, letting user focus on current hint only.

**Review by lancatlin — CHANGES_REQUESTED** [review]
Please rebase main as well

**Comment by Annie04082020** [comment]
rebase branch is opened in another pull request

**Pull request closed without merging**